src/net.cpp

```diff
@@ -2884,3 +2883,23 @@ uint64_t CConnman::CalculateKeyedNetGroup(const CAddress& ad) const
 
     return GetDeterministicRandomizer(RANDOMIZER_ID_NETGROUP).Write(vchNetGroup.data(), vchNetGroup.size()).Finalize();
 }
+
+void LogMessage(const CNode& node, const std::string& msg_type, const Span<const unsigned char>& data, bool is_incoming)
```

Comment:
To make it easier to reason about this function (and its locking requirements) from looking at the function signature, what about replacing `const CNode& node` with `const CAddress& address` since that is what is actually used (`node.addr`)?

Reply:
I like this, thanks.  One day LogMessage may use other attributes of a CNode, but it can be easily changed back when the time comes.  Done.